doc/rvv-intrinsic-spec.adoc

```diff
@@ -614,8 +614,22 @@ The `vlmul_ext` intrinsics are syntactic sugars that have the same semantic as `
 
 The `vcreate` intrinsics are syntactic sugars for tuple types creation. They have the same semantic as multiple `vset`-s filling in values accordingly into the tuple type.
 
+[[pseudo-vlenb]]
+=== `vlenb`
+
+The `vlenb` intrinsic returns what is held inside the read-only CSR `vlenb` ^29^, which is the vector register length in bytes.
+
+[,c]
+----
+unsigned vlenb();
+----
+
 == Programming Notes
 
+=== Semantic of assignment (`=`) in the C language
+
+Without optimizations, the compiler is conservative and assumes an object of RVV type to represent an allocated space on the stack. The assignment operator (`=`) represents a whole register store for the object on the right hand side, and a load for the object on the left hand side.
```

Comment:
The load on the right hand side isn't quite correct. Consider

```
a = vadd(b, c)
```

There are 2 loads on the right side for b and c, and they both occur before the vadd. Not because of the `=`.

Reply:
Improved the description by discussing what to do when users want to copy vector register group contents, which is what we really wanted to clarify.